### ISSUE TITLE
[Bug]: what...the... what is netclient 0.18.0??...

### ISSUE THREAD

**Issue by yura-love** [gravitl/netmaker#1898]
### Contact Details

[Deleted]

### What happened?

When I executed `apt-get upgrade`. node is down.
Sometimes I use the 'upgrade command' to keep dependencies up to date.
None of the binary commands worked, so I checked the version and found that it was 0.18.0. what the... what happened??
Since all the nodes have been upgraded, the network is lost,
so I reopen firewall one by one and manually downgrade.
It's so painful.

```sh
apt-get install netclient=0.17.1-0
```

### Version

v0.17.1

### What OS are you using?

Linux

### Relevant log output

```shell
Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
The following packages were automatically installed and are no longer required:
  libfwupdplugin1 libxmlb1
Use 'sudo apt autoremove' to remove them.
The following packages will be upgraded:
  netclient
1 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/1216 B of archives.
After this operation, 0 B of additional disk space will be used.
(Reading database ... 130300 files and directories currently installed.)
Preparing to unpack .../netclient_0.18.0-0_arm64.deb ...
Unpacking netclient (0.18.0-0) over (0.17.1-0) ...
Setting up netclient (0.18.0-0) ...
```

### Contributing guidelines

- [X] Yes, I did.

**Comment by theguy951357** [comment]
Thank you for letting us know. We are rolling that back to v0.17.1.

**Comment by JeffMatson** [comment]
Just happened to me when setting up a new node. Fresh Debian 11 VM. amd64.

My troubleshooting logs in case it helps are as follows:

TLDR: the package is placing a zero byte binary instead of the netclient bin.

Install log:
```
debian@MariaDB0:~$ sudo apt install netclient
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following NEW packages will be installed:
  netclient
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 1,216 B of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 https://apt.netmaker.org stable/main amd64 netclient amd64 0.18.0-0 [1,216 B]
Fetched 1,216 B in 0s (13.2 kB/s)
Selecting previously unselected package netclient.
(Reading database ... 37484 files and directories currently installed.)
Preparing to unpack .../netclient_0.18.0-0_amd64.deb ...
Unpacking netclient (0.18.0-0) ...
Setting up netclient (0.18.0-0) ...
```

Service status:
```
debian@MariaDB0:~$ sudo systemctl status netclient
● netclient.service - Netclient Daemon
     Loaded: loaded (/lib/systemd/system/netclient.service; enabled; vendor preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Wed 2023-01-04 08:38:00 EST; 12s ago
       Docs: https://docs.netmaker.org
             https://k8s.netmaker.org
    Process: 1487 ExecStart=/sbin/netclient daemon (code=exited, status=203/EXEC)
   Main PID: 1487 (code=exited, status=203/EXEC)
        CPU: 2ms

Jan 04 08:38:00 MariaDB0 systemd[1]: netclient.service: Failed with result 'exit-code'.
```

Checking journalctl:
```
Jan 04 08:13:41 MariaDB0 systemd[3931]: netclient.service: Failed to execute /sbin/netclient: Exec format error
Jan 04 08:13:41 MariaDB0 systemd[3931]: netclient.service: Failed at step EXEC spawning /sbin/netclient: Exec format error
Jan 04 08:13:41 MariaDB0 systemd[1]: netclient.service: Main process exited, code=exited, status=203/EXEC
Jan 04 08:13:41 MariaDB0 systemd[1]: netclient.service: Failed with result 'exit-code'.
Jan 04 08:13:56 MariaDB0 systemd[1]: netclient.service: Scheduled restart job, restart counter is at 5.
```

"exec format error"? Service file looks fine:
```
debian@MariaDB0:~$ sudo cat /lib/systemd/system/netclient.service
[Unit]
Description=Netclient Daemon
Documentation=https://docs.netmaker.org https://k8s.netmaker.org
After=network-online.target
Wants=network-online.target

[Service]
User=root
Type=simple
ExecStart=/sbin/netclient daemon
Restart=on-failure
RestartSec=15s

[Install]
WantedBy=multi-user.target
```

So I checked the binary and found the culprit:
```
debian@MariaDB0:~$ sudo ls -lah /sbin/netclient
-rwxr-xr-x 1 root root 0 Jan  3 15:24 /sbin/netclient
```

Seems the binary is empty in this 0.18.9 deb package.

**Comment by mattkasun** [comment]
repo has been reverted;   problem was result of adding new nmctl v0.18.0 to repo and finger problems
